awx/api/serializers.py

```diff
@@ -4687,7 +4687,8 @@ def _summarize_parent_ujt(self, obj, fk, summary_fields):
         if fk not in summary_keys:
             return
         related_obj = getattr(obj, summary_keys[fk], None)
-        summary_fields[get_type_for_model(related_obj)] = []
+        if related_obj is not None:
```

Comment:
it feels like this block can be folded into the identical check on line 4693/4694